### ISSUE TITLE
Shifter Zerus transforming into Sea Reaver miscellaneous bugs

### ISSUE THREAD

**Issue by Patashu** [HearthSim/hs-bugs#520]
(Presumably it happens with Flame Leviathan too, but that card is not in Standard.)

1) If Shifter Zerus transforms into Sea Reaver, the animation can be seen and an entry is made in the sidebar.

* https://www.reddit.com/r/hearthstone/comments/4gm08h/shifter_zerus_bug/

2) If Shifter Zerus transforms into Sea Reaver than into something else and is played, the displayed card will be a hybrid of the two different card texts.

* https://www.reddit.com/r/hearthstone/comments/4hodx5/shifter_zerus_bug/
* http://us.battle.net/hearthstone/en/forum/topic/20743735796

**Comment by culinko** [comment]
Another report: https://www.reddit.com/r/hearthstone/comments/4ilv53/he_doesnt_even_know_how_to_get_into_play/.

**Comment by culinko** [comment]
Is this a general bug or does Dark Peddler need to be present? Found another report of this: https://www.reddit.com/r/hearthstone/comments/4ki6jr/i_found_the_true_master_of_disguise/.

**Comment by Patashu** [comment]
You're right. It might require Dark Peddler, since I got Sea Reaver from normal Shifter Zerus without a bug while fighting you.

**Comment by Patashu** [comment]
Another report:

https://www.reddit.com/r/hearthstone/comments/4m13np/dr_boom_was_nothing_compared_to_this/

**Comment by Patashu** [comment]
Confirmed in 5.0.0.13 http://puu.sh/peemM.png

**Comment by culinko** [comment]
This might be related, Shifter Zerus itself having its own text, but the image of the minion it transformed into:
* https://www.reddit.com/r/hearthstone/comments/4qj75n/jungle_panther_legendary_confirmed_funny_shifter/
* https://www.reddit.com/r/hearthstone/comments/4qgvso/interesting_shifter_zerus_visual_bug/

**Comment by Patashu** [comment]
Confirmed in 6.1.14830:

https://www.reddit.com/r/hearthstone/comments/58dhlm/shifter_zerus_glitch/
